crates/y-sweet/src/cli.rs

```diff
@@ -7,20 +7,26 @@ pub fn print_server_url(auth: Option<&Authenticator>, url_prefix: Option<&Url>,
     let mut url = if let Some(url_prefix) = url_prefix {
         url_prefix.clone()
     } else {
-        Url::parse(&format!("http://{}", addr)).unwrap()
+        Url::parse(&format!("ys://{}", addr)).unwrap()
     };
 
+    if url.scheme() == "http" {
+        url.set_scheme("ys").unwrap();
+    } else if url.scheme() == "https" {
+        url.set_scheme("yss").unwrap();
+    }
+
     if let Some(auth) = auth {
         url.set_username(&auth.server_token()).unwrap();
     }
 
     let token = url.to_string();
 
-    println!("Use the following server token to connect to y-sweet:");
+    println!("Use the following connection string to connect to y-sweet:");
     println!();
     println!("   {}", token.bright_purple());
     println!();
-    println!("For example, the y-sweet examples expect this parameter as an environment variable:");
+    println!("For example, the y-sweet examples expect this connection string as an environment variable:");
     println!();
     println!("    cd examples/");
     println!("    Y_SWEET_CONFIG={} npm run dev", token.bright_purple());
```

Comment:
thoughts on a better name for this env var? `Y_SWEET_CONNECTION`?

Reply:
`YS_CONNECTION_STRING`? `Y_SWEET_CONNECTION_STR`?